lib/census_api.rb

```diff
@@ -40,6 +40,8 @@ def gender
         "male"
       when "Mujer"
         "female"
+      when "Otro"
```

Comment:
This part is actually a hack which only works for certain Spanish census APIs :sweat_smile:, so I'm not sure which word we should use here :thinking: (if any). @taitus Any ideas?

Reply:
It is not clear to me that we need to add this option until some institution using the Census connection includes this new type of "gender description" in their API response. 🤔 
However, if we want consistent with the other changes of this PR, the word "other" seems right to me.